src/store/single.rs

```diff
@@ -128,7 +128,7 @@ impl<'i, I> Iterator for Iter<'i, I>
 where
     I: BackendIter<'i>,
 {
-    type Item = Result<(&'i [u8], Option<Value<'i>>), StoreError>;
+    type Item = Result<(&'i [u8], Value<'i>), StoreError>;
```

Comment:
I like this change👍, it solves [this itch](https://github.com/mozilla/rkv/pull/35#issuecomment-395879259) for me :)

We need to at least bump the minor version as this breaks the old iterator API.